python_modules/dagster/dagster_tests/core_tests/test_logging.py

```diff
@@ -364,3 +365,42 @@ def logged_solid(context):
 
     assert re.search("test OUTPUT debug logging from logged_solid.", captured.err, re.MULTILINE)
     assert re.search("test INPUT debug logging from logged_solid.", captured.err, re.MULTILINE)
+
+
+def test_conf_file_logging(capsys):
+    config_settings = {
+        "python_logs": {
+            "handlers": {
+                "handlerOne": {
+                    "class": "logging.StreamHandler",
+                    "level": "INFO",
+                    "stream": "ext://sys.stdout",
+                },
+                "handlerTwo": {
```

Comment:
I think we should have a test that includes a custom handler (i.e. something not defined in the logging module).